File: b1158.py
N, K = map(int, input().split())

circle = [i for i in range(1, N + 1)]
Josephus = []
start_idx = 0
while len(Josephus) < N:
    start_idx = (start_idx + K - 1) % len(circle)
    Josephus.append(circle.pop(start_idx))
print("<{}>".format(", ".join(map(str,Josephus))))

File: b1158.cpp
#include <iostream>
#include <vector>


int main() {
	int N, K;
	std::cin >> N >> K;
	std::vector<int> circle;
	circle.resize(N);
	for (int i = 0; i < N; ++i) {
		circle[i] = i + 1;
	}

	std::vector<int> Josephus;
	int start_idx = 0;
	int interval = K - 1;
	while (Josephus.size() < N) {
		start_idx = (start_idx + interval) % circle.size();
		Josephus.push_back(circle[start_idx]);
		circle.erase(circle.begin() + start_idx);
	}
	
	std::cout << "<";
	for (int i = 0; i < N; ++i) {
		if (i == N - 1) {
			std::cout << Josephus[i] << ">" << std::endl;
			continue;
		}
		std::cout << Josephus[i] << ", ";
	}
}